### PR TITLE
Center the map in mobile mode, resize tweets, fix description, grammar and punctuation.

### LINKED ISSUE #8: content not loading due to mixed content
checkout https://github.com/pythonindia/junction/issues/212

### PR DESCRIPTION
**Pull request by astronomersiva**
Fix the description to "It attracts the best Python programmers **from** across the country and abroad".

Load fonts over https instead of http. Fixes #8.

Center the map in the venue section in the mobile version and add some space between the map and the venue image. Currently, it shows up like this:

image

Resize the Twitter stream box to the same as that of the blog feed.  

Fix grammar and punctuation. 

Change & to `&amp;`

### CODE REVIEW

File: sponsorship.html [hunk 1]
```diff
@@ -254,9 +254,9 @@ <h4 class="text-center"><img class="" src="images/associate_sponsor.png" height=
                     <h4>Note:</h4>
                     <ul>
                         <li>50% discount on silver category of sponsorship for startups with less than 10 employees.</li>
-                        <li>Stall will be available only during main conference days (Saturday & Sunday).</li>
+                        <li>Stall will be available only during the main conference days (Saturday & Sunday).</li>
```
Comment (shrayasr) [inline sponsorship.html]: While you're at it, fix the `&` to `&amp;` ?

Comment (astronomersiva) [inline sponsorship.html]: Done :+1:

File: sponsorship.html [hunk 2]
```diff
@@ -26,7 +26,7 @@
     <meta name="description" content="PyCon India, the premier conference in India on using and developing the Python programming language is conducted annually by the Python developer community and it attracts the best Python programmers from across the country and abroad.">
     <meta name="author" content="Pycon India">
     <link rel="shortcut icon" href="favicon.png">
-    <link href='http://fonts.googleapis.com/css?family=Lato:300,400,300italic,400italic' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Lato:300,400,300italic,400italic' rel='stylesheet' type='text/css'>
```
Comment (kracekumar) [inline sponsorship.html:29]: Thanks for this. It is better to use `//fonts`

**Comment by kracekumar** [comment]
@astronomersiva :+1: In general if a pull request changes UI, it is good show screenshot before and after the change.

**Comment by astronomersiva** [comment]
@kracekumar Will keep that in my mind. Thanks :smile:

**Pull request merged**